.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -1,6 +1,9 @@
 # Ref: https://help.github.com/en/github/building-a-strong-community/configuring-issue-templates-for-your-repository#configuring-the-template-chooser
 blank_issues_enabled: false  # default is true
 contact_links:
+- name: Python Infrastructure Status Page
+  url: https://status.python.org
+  about: Please check this page, prior to reporting possible PyPI outages
```

Comment:
*(I wasn't sure about this text, feel free to improve it, the PR allows repo maintainers to update the branch)*